plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -31,7 +31,7 @@ import React, { useCallback, useMemo, useState } from 'react';
 import { Link as RouterLink } from 'react-router-dom';
 import { catalogApiRef } from '../../api/types';
 import { EntityFilterGroupsProvider, useFilteredEntities } from '../../filter';
-import { useStarredEntities } from '../../hooks/useStarredEntites';
```

Comment:
ha, nice work

Reply:
this was totally my spelling of entities.

Reply:
The vscode spell check extension is great! It's what showed this to me, easy to miss